### PR TITLE
chore(CPlusPlus): add remove occurence of characters of 2nd string from the 1st string

### PR DESCRIPTION
**Pull request by samughodake**
**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [x] New algorithm
- [ ] Optimization in previous algorithms
- [ ] Code style update
- [ ] Refactor
- [ ] Documentation
- [ ] Other, please describe:

**Briefly describe the changes in this PR**

**Other information:**

### CODE REVIEW

**Comment by ming-tsai** [comment]
Hi @samughodake,
Does this pull request is the same change for #468?

File: algorithms/CPlusPlus/Strings/remove-occurences.cpp [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+*/
+
+#include <string>
+#include <iostream>
+using namespace std;
+
+string removeChars(string s1, string s2) {
+    
+    string out;
+    //checking if the character of 2nd string is present in the 1st string
```
Comment (Ashborn-SM) [inline algorithms/CPlusPlus/Strings/remove-occurences.cpp:13]: I believe you mean `checking the characters of the 1st string is present in the 2nd string`.
In line 15 you are looping through `s1` and in line 17 you are checking if the characters of 1st string is present
in `s2`, if you interchange those variables the end result will be different.
```suggestion
    //checking if the character of the 1st string is present in the 2nd string
```

**Review by Ashborn-SM — CHANGES_REQUESTED** [review]
@samughodake you can tell me if i am mistaken.

**Review by Ashborn-SM — APPROVED** [review]

**Comment by samughodake** [comment]
Thanks a lot! @Ashborn-SM 
should I update the readme file now?

**Comment by Ashborn-SM** [comment]
> Thanks a lot! @Ashborn-SM
> should I update the readme file now?

Yes, totally forgot about that mb

**Comment by samughodake** [comment]
> > Thanks a lot! @Ashborn-SM
> > should I update the readme file now?
> 
> Yes, totally forgot about that mb

will update the readme,thanks.

**Review by Ashborn-SM — APPROVED** [review]
Good job!!

**Comment by samughodake** [comment]
> Good job!!

Thanks. Please merge this branch

**Review by samughodake — COMMENTED** [review]
Thanks for fixing typo errors

**Review by Ashborn-SM — APPROVED** [review]

**Pull request merged**